dhis-2/dhis-api/src/main/java/org/hisp/dhis/category/CategoryOptionGroupSet.java

```diff
@@ -69,7 +70,9 @@ public CategoryOptionGroupSet()
 
     public CategoryOptionGroupSet( String name )
     {
+
         this.name = name;
+        this.shortName = name;
```

Comment:
This one might be a bit confusing and might lead to issues later? Perhaps better to add another constructor?

Reply:
We could add a `shortName` parameter to this one and add a value everywhere it is used. I just thought since this is only used in tests this would cause unnecessary many changes. It would be nice if we had a clear and agreed way of marking constructors that should only be used from tests then it would be more clear if this is a problem or not.

Reply:
Right. I was just thinking since we don't have that now, then it may lead to confusion later if someone else starts to use the constructor. Anyway, not a major point.